src/main/resources/config/application-core.yml

```
@@ -9,6 +9,11 @@ artemis:
         enabled: true
     tutorialgroup:
         enabled: true
+    nebula:
+        enabled: false
+
+
+
     user-management:
```

```suggestion
    nebula:
        enabled: false

user-management:
```

Also, if you always default to false here (as you should), you could consider just putting it in, e.g., application-artemis.yml.